### PR TITLE
Adding FR.fr

### PR DESCRIPTION
**Pull request by yezz123**
## Track translation progress by opening a draft PR using this template and checking off the translations completed

Each lesson includes a translation of the README.md and the Assignment.md file, if available. Only mark the lesson complete if both those files are translated per lesson, please.

- [x] 1
- [X] 1-1
- [X] translations

### CODE REVIEW

**Comment by jlooper** [comment]
aha! bonjour!

**Comment by yezz123** [comment]
> aha! bonjour!

Bonjour! @jlooper

**Review by jlooper — COMMENTED** [review]
hi, do you want me to merge this? will you continue doing the other files? Merci!

**Comment by yezz123** [comment]
> hi, do you want me to merge this? will you continue doing the other files? Merci!

Bonsoir @jlooper !
You can merge it and when i finish the others i will create an other Pull request with the new Files.
Maybe also i will try to translate it to AR.ar (Arabic language).

File: 1-getting-started-lessons/1-intro-to-programming-languages/translations/README.fr.md [hunk 1]
```diff
@@ -0,0 +1,191 @@
...
+Dans cette leçon, nous aborderons:
+
+- Qu'est-ce que la programmation?
+- Types de langages de programmation
+- Éléments de base d'un programme
+- Logiciel et outillage utiles pour le développeur professionnel
+
+## Qu'est-ce que la programmation?
+
+PLa programmation (également appelée codage) est le processus d'écriture d'instructions sur un appareil, tel qu'un ordinateur ou un appareil mobile. Nous écrivons ces instructions avec un langage de programmation, qui est ensuite interprété par l'appareil. Ces ensembles d'instructions peuvent être désignés sous différents noms, mais *programme*, *programme informatique*, *application (application)* et *exécutable* sont quelques noms courants.
```
Comment (bl-ue) [inline 1-getting-started-lessons/1-intro-to-programming-languages/translations/README.fr.md:22]:
```suggestion
Pla programmation (également appelée codage) est le processus d'écriture d'instructions sur un appareil, tel qu'un ordinateur ou un appareil mobile. Nous écrivons ces instructions avec un langage de programmation, qui est ensuite interprété par l'appareil. Ces ensembles d'instructions peuvent être désignés sous différents noms, mais *programme*, *programme informatique*, *application (application)* et *exécutable* sont quelques noms courants.
```

File: 1-getting-started-lessons/1-intro-to-programming-languages/translations/README.fr.md [hunk 2]
```diff
@@ -0,0 +1,191 @@
...
+
+✅ Faites une petite recherche: qui est considéré comme le premier programmeur informatique au monde?
+
+## Langages de programmation
+
+Les langages de programmation ont un objectif principal: permettre aux développeurs de créer des instructions à envoyer à un appareil. Les appareils ne peuvent comprendre que le binaire (1 et 0), et pour *la plupart* les développeurs, ce n'est pas un moyen très efficace de communiquer. Les langages de programmation sont un vecteur de communication entre les humains et les ordinateurs.
+
+Les langages de programmation se présentent sous différents formats et peuvent servir à des fins différentes. Par exemple, JavaScript est principalement utilisé pour les applications Web, tandis que Bash est principalement utilisé pour les systèmes d'exploitation.
+
+* Les langues de bas niveau *nécessitent généralement moins d'étapes que les* langues de haut niveau * pour qu'un appareil interprète les instructions. Cependant, ce qui rend les langages de haut niveau populaires, c'est leur lisibilité et leur support. JavaScript est considéré comme un langage de haut niveau.
```
Comment (bl-ue) [inline 1-getting-started-lessons/1-intro-to-programming-languages/translations/README.fr.md:34]:
```suggestion
*Les langues de bas niveau* nécessitent généralement moins d'étapes que les *langues de haut niveau* pour qu'un appareil interprète les instructions. Cependant, ce qui rend les langages de haut niveau populaires, c'est leur lisibilité et leur support. JavaScript est considéré comme un langage de haut niveau.
```

**Review by bl-ue — CHANGES_REQUESTED** [review]
Hi! I don't know French, but I found some apparent mistakes.

File: 1-getting-started-lessons/1-intro-to-programming-languages/translations/README.fr.md (hunk 2 above)
Comment (bl-ue) [reply 1-getting-started-lessons/1-intro-to-programming-languages/translations/README.fr.md:34]: In this one it looks like the emphasis characters were pushed around...

**Review by bl-ue — APPROVED** [review]

**Comment by yezz123** [comment]
> Hi! I don't know French, but I found some apparent mistakes.

Thank you @bl-ue for your reviews i appreciate that 🥇

**Comment by jlooper** [comment]
hi, can I get an idea of status here?

**Comment by yezz123** [comment]
> hi, can I get an idea of status here?

The status stay stable no updates just the same file 👍🏻

**Comment by jlooper** [comment]
> > hi, can I get an idea of status here?
> 
> The status stay stable no updates just the same files 👍🏻

Thanks, I will wait until the task is complete, in this case, to avoid partial translations. thank you!

**Review by manman1500 — COMMENTED** [review]
0991615378

**Comment by manman1500** [comment]
0991615378

**Review by manman1500 — APPROVED** [review]

**Comment by jishnugit** [comment]
9567889784

**Comment by milliondollarbabe11** [comment]
thanks I am good

**Comment by bl-ue** [comment]
@yezz123 this is done, right?  If so, @jlooper can we get it merged and then lock it to stop this odd spam?

**Comment by yezz123** [comment]
> @yezz123 this is done, right? If so, @jlooper can we get it merged and then lock it to stop this odd spam?

I guess its done but i will updates it anytime if i got time.
But for those file are good & clean translate 👍🏻

**Comment by jlooper** [comment]
yes, the spam is EXTREMELY annoying @manman1500 and needs to stop

I'll merge these, thank you @yezz123

**Pull request merged**